### PR TITLE
Added/Corrected a few item_mods

### PR DESCRIPTION
**Pull request by Ninjistix**
added some item mods

### CODE REVIEW

File: sql/item_mods.sql [hunk 1]
```diff
@@ -35278,20 +35470,20 @@ INSERT INTO `item_mods` VALUES (28154,29,5);
...
-INSERT INTO `item_mods` VALUES (28155,13,16);
-INSERT INTO `item_mods` VALUES (28155,14,12);
-INSERT INTO `item_mods` VALUES (28155,23,28);
-INSERT INTO `item_mods` VALUES (28155,29,3);
-INSERT INTO `item_mods` VALUES (28155,31,75);
-INSERT INTO `item_mods` VALUES (28155,68,22);
-INSERT INTO `item_mods` VALUES (28155,162,-8);
-INSERT INTO `item_mods` VALUES (28155,288,2);
-INSERT INTO `item_mods` VALUES (28155,384,51);
+INSERT INTO `item_mods` VALUES (28155,1,66); -- Scuffler's cosciales(Base DEF) 66
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: Base def 66 ?

I see def 123 on the items helptext.

**Comment by Ninjistix** [comment]
@TeoTwawki fixed the wrong base def on 28155

**Comment by Ninjistix** [comment]
should i close this? not sure if its out dated... because it say's there's conflicts now

**Comment by TeoTwawki** [comment]
Rebase it onto latest and check for duplicates

**Comment by Ninjistix** [comment]
updated it.. took awhile because i don't have access to this pc as much.

**Comment by TeoTwawki** [comment]
I think that was someone trying to make the comments aligned using multiple spaces actually (those aren't hard tabs in the diff). Its fine in either format either space aligned or one-spaced. Hard tabs show up as a large right facing arrow, spaces show as centered dots.

**Comment by Ninjistix** [comment]
they were dots on my end, just removed the extras

**Comment by TeoTwawki** [comment]
Yeah multi spaces. np with the change.

**Comment by Ninjistix** [comment]
actually i found out what it was, i think its an addon for notepad++ called code alignment? honestly i have no qualms with this, only if the whole file gets converted.. because having half done and half not is annoying..

**Comment by TeoTwawki** [comment]
I hear you. I have a thing about consistency myself.

For other reviewers:
https://github.com/DarkstarProject/darkstar/pull/3457/files?w=1
Is able to show the non whitespace changes .

**Comment by Ninjistix** [comment]
oh cool thanks @TeoTwawki , didn't know about that 👍

**Comment by TeoTwawki** [comment]
I wasn't super thorough but looks good to me.

**Comment by takhlaq** [comment]
no way am i merging changes i can't see
also take another look at gelai earring

**Pull request closed without merging**